Bottom: ea47c4d9f77571571379fc8ed9b035e53d705bdf
Top:    39966e89a3b1be8a42f581aedfbf38af1d1d2335
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-12 09:01:54 -0500

Refresh of all-docker_network

---
diff --git a/ansible-playbook/group_vars/docker-hosts/networking.yml b/ansible-playbook/group_vars/docker-hosts/networking.yml
index 77d9ce6a..42db8fdb 100644
--- a/ansible-playbook/group_vars/docker-hosts/networking.yml
+++ b/ansible-playbook/group_vars/docker-hosts/networking.yml
@@ -1,64 +1,53 @@
 ---
 
-svn_ip_addr: "{{ daedalus_subnet | ipaddr('101') | ipaddr('address') }}"
 svn_hostname: svn.novatech-llc.com
 svn_port_args:
 - "{{ svn_ip_addr }}:80:80"
 - "{{ svn_ip_addr }}:443:443"
 
-gitlab_ip_addr: "{{ daedalus_subnet | ipaddr('102') | ipaddr('address') }}"
 gitlab_hostname: git.novatech-llc.com
 gitlab_port_args:
 - "{{ gitlab_ip_addr }}:80:80"
 - "{{ gitlab_ip_addr }}:443:443"
 - "{{ gitlab_ip_addr }}:22:22"
 
-wiki_ip_addr: "{{ daedalus_subnet | ipaddr('103') | ipaddr('address') }}"
 wiki_hostname: wiki.novatech-llc.com
 wiki_port_args:
 - "{{ wiki_ip_addr }}:80:80"
 - "{{ wiki_ip_addr }}:443:443"
 
-mantisbt_ip_addr: "{{ daedalus_subnet | ipaddr('104') | ipaddr('address') }}"
 mantisbt_hostname: mantis.novatech-llc.com
 mantisbt_port_args:
 - "{{ mantisbt_ip_addr }}:80:80"
 - "{{ mantisbt_ip_addr }}:443:443"
 
-build_ip_addr: "{{ daedalus_subnet | ipaddr('105') | ipaddr('address') }}"
 build_hostname: buildsystem.novatech-llc.com
 build_port_args:
 - "{{ build_ip_addr }}:80:80"
 - "{{ build_ip_addr }}:443:443"
 
-testssh_ip_addr: "{{ daedalus_subnet | ipaddr('106') | ipaddr('address') }}"
 testssh_hostname: testssh.novatech-llc.com
 testssh_port_args:
 - "{{ testssh_ip_addr }}:2200:22"
 
-exim4_ip_addr: "{{ daedalus_subnet | ipaddr('107') | ipaddr('address') }}"
 exim4_hostname: mail.novatech-llc.com
 exim4_port_args:
 - "{{ exim4_ip_addr }}:25:25"
 - "{{ exim4_ip_addr }}:465:465"
 - "{{ exim4_ip_addr }}:587:587"
 
-openldap_ip_addr: "{{ daedalus_subnet | ipaddr('108') | ipaddr('address') }}"
 openldap_hostname: ldap.novatech-llc.com
 openldap_port_args:
 - "{{ openldap_ip_addr }}:389:389"
 - "{{ openldap_ip_addr }}:636:636"
 
-phpldapadmin_ip_addr: "{{ daedalus_subnet | ipaddr('108') | ipaddr('address') }}"
 phpldapadmin_hostname: ldap.novatech-llc.com
 phpldapadmin_port_args:
 - "{{ phpldapadmin_ip_addr }}:80:80"
 - "{{ phpldapadmin_ip_addr }}:443:443"
 
-phpmyadmin_ip_addr: "{{ daedalus_subnet | ipaddr('109') | ipaddr('address') }}"
 phpmyadmin_hostname: phpmyadmin.novatech-llc.com
 
-buildbot_ip_addr: "{{ daedalus_subnet | ipaddr('110') | ipaddr('address') }}"
 buildbot_hostname: buildbot.novatech-llc.com
 buildbot_port_args:
 - "{{ buildbot_ip_addr }}:80:8080"
diff --git a/ansible-playbook/host_vars/daedalus.novatech-llc.com/networking.yml b/ansible-playbook/host_vars/daedalus.novatech-llc.com/networking.yml
index f567d255..5e2d2774 100644
--- a/ansible-playbook/host_vars/daedalus.novatech-llc.com/networking.yml
+++ b/ansible-playbook/host_vars/daedalus.novatech-llc.com/networking.yml
@@ -1,4 +1,18 @@
 ---
 
+ansible_user: ansibleremote
+
 daedalus_subnet: 172.16.0.0/16
 daedalus_net_connection: "Bridge br0"
+
+svn_ip_addr:          172.16.0.101
+gitlab_ip_addr:       172.16.0.102
+wiki_ip_addr:         172.16.0.103
+mantisbt_ip_addr:     172.16.0.104
+build_ip_addr:        172.16.0.105
+testssh_ip_addr:      172.16.0.106
+exim4_ip_addr:        172.16.0.107
+openldap_ip_addr:     172.16.0.108
+phpldapadmin_ip_addr: 172.16.0.108
+phpmyadmin_ip_addr:   172.16.0.109
+buildbot_ip_addr:     172.16.0.110
diff --git a/ansible-playbook/host_vars/testdaedalus.novatech-llc.com/networking.yml b/ansible-playbook/host_vars/testdaedalus.novatech-llc.com/networking.yml
index 5111648b..aa95e93b 100644
--- a/ansible-playbook/host_vars/testdaedalus.novatech-llc.com/networking.yml
+++ b/ansible-playbook/host_vars/testdaedalus.novatech-llc.com/networking.yml
@@ -1,4 +1,18 @@
 ---
 
-daedalus_subnet: 172.16.103.0/16
+ansible_user: ansibleremote
+
+daedalus_subnet: 172.16.0.0/16
 daedalus_net_connection: "enp5s0f0"
+
+svn_ip_addr:          172.16.103.101
+gitlab_ip_addr:       172.16.103.102
+wiki_ip_addr:         172.16.103.103
+mantisbt_ip_addr:     172.16.103.104
+build_ip_addr:        172.16.103.105
+testssh_ip_addr:      172.16.103.106
+exim4_ip_addr:        172.16.103.107
+openldap_ip_addr:     172.16.103.108
+phpldapadmin_ip_addr: 172.16.103.108
+phpmyadmin_ip_addr:   172.16.103.109
+buildbot_ip_addr:     172.16.103.110
diff --git a/ansible-playbook/site.yml b/ansible-playbook/site.yml
index 2f70867f..de8660ab 100644
--- a/ansible-playbook/site.yml
+++ b/ansible-playbook/site.yml
@@ -60,7 +60,7 @@
   roles:
   - name: Configure Docker networking
     role: docker-networking
-    net_connection: '{{ daedalus_net_connection }}'
+    network_connection: '{{ daedalus_net_connection }}'
     subnet: '{{ daedalus_subnet }}'
     services:
     - svn
diff --git a/roles/docker-networking/tasks/ip_addr.yml b/roles/docker-networking/tasks/ip_addr.yml
index f5d4ff8b..5ef28e7f 100644
--- a/roles/docker-networking/tasks/ip_addr.yml
+++ b/roles/docker-networking/tasks/ip_addr.yml
@@ -23,9 +23,9 @@
   command: >
     nmcli connection 
     modify '{{ network_connection }}' 
-    +ipv4.addresses '{{ ip_addr }}/{{ subnet | ipaddr("prefix") }}'
+    +ipv4.addresses '{{ ip_addr }}'
   vars:
-    ip_addr: '{{ lookup("vars", item + "_ip_addr") }}'
+    ip_addr: '{{ lookup("vars", item + "_ip_addr") }}/{{ subnet | ipaddr("prefix") }}'
   loop: "{{ services }}"
  
 - sysctl:
